testslide/__init__.py

```diff
@@ -56,36 +71,36 @@ class _ContextData(object):
     example execution.
     """
 
-    def _init_sub_example(self):
+    def _init_sub_example(self) -> None:
         self._sub_examples_agg_ex = AggregatedExceptions()
 
-        def real_assert_sub_examples(self):
+        def real_assert_sub_examples(self: "_ContextData") -> None:
             if self._sub_examples_agg_ex.exceptions:
                 self._sub_examples_agg_ex.raise_correct_exception()
 
         if self._example.is_async:
 
-            async def assert_sub_examples(self):
+            async def assert_sub_examples(self: "_ContextData") -> None:
                 real_assert_sub_examples(self)
 
         else:
 
-            def assert_sub_examples(self):
+            def assert_sub_examples(self: "_ContextData") -> None:  # type: ignore
```

Comment:
add typing info AND ignore it?

Reply:
lol. thx. Must have been drunk or idk :D

Reply:
it's because in one case it's sync in the other case it's async. So it causes: testslide/__init__.py:89: error: All conditional function variants must have identical signatures